testsuite/tests/apicast/policy/batcher/test_batcher_policy_mapping_rules.py

```diff
@@ -44,7 +44,9 @@ def service(service):
 @pytest.fixture(scope="module")
 def client(api_client):
     """We are testing path that doesn't match mapping rule so we need to disable retry"""
-    assert api_client().get("/get").status_code == 200
+    assert api_client().get("/get").status_code == 200  # Ensures that service is set up correctly
+
+    sleep(50)  # need to wait 50 seconds because of batcher policy to ensure stable environment for tests
```

Comment:
Was this problem before as well, or is the sleep just for persistence?

Reply:
It was a problem before as well, but was resolved by +4 in test assert instead of +3 as it should. The persistence plugin just shows us the problem with this test.